app/javascript/mastodon/locales/pl.json

```diff
@@ -73,7 +73,7 @@
   "emoji_button.search": "Szukaj...",
   "emoji_button.symbols": "Symbole",
   "emoji_button.travel": "Podróże i miejsca",
-  "empty_column.community": "Lokalna oś czasu jest pusta. Napisz coś publicznie, aby odbić piłeczkę!",
+  "empty_column.community": "Lokalna oś czasu jest pusta. Napisz coś publicznie, aby zagaić!",
```

Comment:
Maybe then „aby rozpocząć dyskusję”? However [according to the PWN dictionary, „zagaić” means roughly the same](https://sjp.pwn.pl/sjp/zagaic;2542265.html) (and is shorter). Either the former or the latter is acceptable for me though.